docs/tutorials/misc/300_otcp.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.        label="Contour of Confidence Region",
</pre><p>This can also be removed, it's not being shown.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>